components/StatusItem/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+import { Constants } from 'expo';
+import PropTypes from 'prop-types';
+
+
+class Icon extends Component {
+  render() {
+    if (this.props.danceStatus == 'Uploaded') {
+    pic=require('..\\icons\\checkmark.png')
+  } else {
+    pic=require('..\\icons\\exclamation.jpg')
+  }
+    return (
+      <View style={{flex: 1}} >
+        <Image source={pic} style= {style.icon} />
+      </View>
+    );
+  }
+}
+
+class Information extends Component {
+  render() {
+    return (
+      <View style={style.information}>
+        <Text style={style.danceNum}> # {this.props.danceNum} </Text>
+        <Text style={style.danceName}> {this.props.danceName} </Text>
```

Comment:
I belieeeve you don't want spaces here surrounding the string you want rendered in this `Text` block. 

For example:
what you're doing is this:
```
<Text> hello world! </Text>   
```
this will render Text ' hello world! ', string with surrounding spaces making spacing like margins on it look jenky

what we want is this:
```
<Text>hello world!</Text>    
```
this will render Text 'hello world!', correct string with no surrounding spaces